_docs/applications/wordpress.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: WordPress Demo
+title: WordPress
 anchor: applications
 module: applications/wordpress
 # Last reviewed by cheryl.hung@storageos.com on 2017-04-12 - commands work fine but can't connect to WP
```

Comment:
I've hashed out the menu item for this as I'm not happy with it right now.  There are a few command line changes with Swarm and I can't see the storage getting provisioned through docker volume ls